### PR TITLE
[W6.4b][T16-B2]Lin Jiachao

### PR DESCRIPTION
**Pull request by lincredibleJC**
Add NFRs to the DeveloperGuide.md

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be carry out operations within O(log N) time
```
Comment (brandonyeoxg) [inline doc/DeveloperGuide.md:97]: Recall from tutorial that NFRs should be in layman terms, the people reading the NFRs may not know what is `O(lg N)` time =).

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,12 @@ Use case ends.
...
+6. Should have an intuitive user interface
```
Comment (brandonyeoxg) [inline doc/DeveloperGuide.md:98]: Recall from tutorial that NFRs are constraints and therefore needs to be quantified, can this be quantified?

**Review by brandonyeoxg — COMMENTED** [review]
Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail.

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (lincredibleJC) [reply doc/DeveloperGuide.md:97]: Yes, so something  along the lines of:
5. Should respond to user input within 1 second.
Is that acceptable?

File: doc/DeveloperGuide.md (hunk 2 above)
Comment (lincredibleJC) [reply doc/DeveloperGuide.md:98]: So NFR 6 should be
6. Should have the relevant logos on buttons to tell what the button does.
Is that okay?
